WordPress/Classes/Networking/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -118,4 +208,35 @@ - (RemotePostTag *)remoteTagFromXMLRPCDictionary:(NSDictionary *)xmlrpcDictionar
     return tag;
 }
 
+- (NSDictionary *)parametersForPaging:(RemoteTaxonomyPaging *)paging
```

Comment:
This looks identical to the REST version.  Maybe its a good candidate to extract into a helper class vs duplication?